tss-esapi/tests/integration_tests/structures_tests/pcr_tests/pcr_select_size_tests.rs

```diff
@@ -121,3 +128,39 @@ fn test_default() {
         "PcrSelectSize did not have the expected default value",
     );
 }
+
+#[test]
+fn test_try_parse_u8_with_invalid_values() {
+    for value in bad_u8_values() {
+        assert_eq!(
+            Err(Error::WrapperError(WrapperErrorKind::InvalidParam)),
+            PcrSelectSize::try_parse_u8(value),
+            "try_parse_u8 using an invalid size_of_select value{} did not result in the expected error",
```

Comment:
Hmm... a space between `value` and `{}` is missing?

Reply:
Fixed